### PR TITLE
Added prop for passing config object to babel

### PR DESCRIPTION
**Pull request by jquense**
Allows configuring specific features that should or should not be
transpiled, and lets you enable stage: 0 features like class property
initializers

### CODE REVIEW

**Comment by kenwheeler** [comment]
Looks great, let me test this before merging

**Comment by jquense** [comment]
Awesome! and thanks again for the excellent component

File: README.md [hunk 1]
```diff
@@ -151,3 +151,14 @@ Turns preview into a simple console for testing out ES6 code. Use `console.log()
   es6Console={true}
   codeText={es6Example} />
 ```
+
+###babelConfig
+_React.PropTypes.object_
+
+An object of [configuration options](http://babeljs.io/docs/usage/options/) that the playground should pass to babel. Use to adjust, or limit the features you want the playground to support.
```
Comment (SoGood13) [inline README.md]: <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-55660b4008d9c380" async="async"></script>

**Comment by jquense** [comment]
any chance any of these will ever get merged? No rush of course, but we are trying to figure out whether to just publish our forked version.

**Pull request closed without merging**